script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -571,109 +576,108 @@ I have no mental capacity left to allocate to any other eventualities.
 [sha:0df06b142239c91ae758079373e614b3eefa6017]{
 -- Page 693, Offset 10528.
 -- だからもう、降り始めた俺には<対|も><応|の>を考える機能はない。
-So now that I've started to descend,
- I don't have the ability to think about how to respond to him.
+Now that I've initiated the descent,
+ I've lost the ability to formulate thoughts about how to react to him.
 }
 [sha:c057faa764c9a44cf6a342af0afa3732afa36dc0]{
 -- Page 694, Offset 10529.
 -- もしマイナスが二ヵ所に留まらない場合は？
-// "in that period" added to clarify that the problem is his clock-rate and the
-// limited time period
-What if I have to handle more than just two of those do-or-die moments in that period?
+What if there are more than two of these critical junctures?
 }
 [sha:a246cbce56656d812d3d22aa8c69c334bdbeef13]{
 -- Page 694, Offset 10530.
 -- それは考慮しない。三ヵ所もの『死』を超える余力は俺にはない。その時はヤツの勝ちだ。ヤツの異常性が、俺の想像を上回っているだけの話。
-I do not consider that.
- I don't have the capacity to overcome three 'deaths' in my mind.
+I won't consider it.
+ I don't have it in me to overcome three of them.
  If that's how it plays out, victory is his.
- It would just mean that his power is stronger than I can imagine.
+ That would just mean his powers were stronger than I imagined.
 }
 [sha:75680ba2b152fc7b3c04412eff6c4bb7e01ba93f]{
 -- Page 695, Offset 10531.
 -- 氷の槍は折り重なるように、落下する獲物に殺到する。
-A dense flurry of ice spears shoots toward the falling prey.
+A dense flurry of ice spears bombard the falling prey.
 }
 [sha:ba9c2a1086ba55450f0e1c97fe70b959fa0c1def]{
 -- Page 695, Offset 10532.
 -- まるで獣の<顎|あぎと>だ。恐ろしいがどうでもいい。
-It's like the jaws of a beast. It's terrifying, but it doesn't matter.
+Not unlike the jaws of a beast. Terrifying, but inconsequential.
 }
 [sha:9f532dbc35cebc5dc45cc94b45e9300a64065ff8]{
 -- Page 695, Offset 10533.
 -- そんな、ただ本能に因っただけの<咀嚼|そしゃく>など知った事か。
-I wonder if it's thanks to his instincts that he anticipated this.
+I don't give a damn about such an instinctual display of voraciousness.
 }
 [sha:0032cdf383b9b0e7c3f16d1068f5192c946dc6f9]{
 -- Page 695, Offset 10534.
 -- 俺がそうくると確信し、そうなれば殺されると恐れた迎撃は、
 Convinced that I would come for him,
- and terrified that I would kill him when I did...
+ and terrified that I would kill him when I did, he counterattacks―――
 }
 [sha:a7dd57312c121a7f9c51ed5e3e188e8021d13109]{
 -- Page 696, Offset 10535.
 -- ―――10メートル。
-―――Ten meters.
+―――At 10 meters.
 }
 [sha:4588bd93b586acd91c5ad6c79553cf6c46f49156]{
 -- Page 696, Offset 10536.
 -- 一つめの<致|マ><死|イ><地|ナ><点|ス>。
-// maybe 'decision' with ruby as 'chance for failure'?
-The first chance for failure.
+The first critical juncture comes.
 }
 [sha:14ddcf60c129574c8c7ce8d52a5e0be55c98fe41]{
 -- Page 696, Offset 10537.
 -- 手足の感覚、筋力の微調整、体幹の急制動とその返し。
-The sensations of my limbs,
- the fine-tuning of my muscles,
- and the sudden braking of my torso return.
+I sense my limbs move,
+ feel them adjust the movement of my muscles.
+ My torso make a sudden brake, and then flips.
```

Comment:
makes

Reply:
I feel like 体幹の急制動とその返し is him whipping his torso one way then the other to shift his weight (alter trajectory), I don't feel like he's doin flips and shit on the boulder
> I sense the weight of my own limbs, fine tune the strength I'm putting into my muscles, then suddenly jerk my torso to the side then back again.

Reply:
Yeah, that might be too complicated.
It has to lead into the next sentence, too.
For reference:
>After witnessing his angle of fire,
> I alter the natural course of my trajectory!

So I think something like this might be preferrable. 
I feel the muscles in my limbs adjust.
 My torso make a sudden brake, and then bends in the opposite direction.

Reply:
I think that still doesn't really convey what's going on well? The suggestion leads in fine to the next sentence imo. I'd also keep it active over passive either way.

Reply:
I kinda like the passive stuff (which I usually don't) because it gives the impression of him working on instinct.

Reply:
Fair, but still think e.g.
> I feel the muscles in my limbs adjust. My torso suddenly whips to one side, then back again.

is a clearer descr of what (I think) is happening here